### DISCUSSION TITLE
Copyright?

### DISCUSSION THREAD

**Discussion by olikraus** [robhagemans/hoard-of-bitfonts#29] [Q&A]
Hi All

Very nice collection. However I wonder whether there is any copyright hint for the fonts.
Any idea? Did I miss the copyright remarks somewhere?

Thanks

**Comment by robhagemans** [comment]
Hi, it depends on your jurisdiction whether bitmap font representations
like these are copyrightable. See the explanation in LICENSE.md.

The original font files generally are copyrighted as software so they are
not included, but for pure bitmaps the situation may be more liberal,
depending on where you're based.

I have retained any copyright and license notices that were present in the
original files. Where they were absent any copyright that applies in your
jurisdiction would belong to the author or issuer of the font.

Hope this helps

**Reply by olikraus** [reply]
Thanks a lot for the clarification. It would be more easier to have a clear statement...
Anyhow. It's a great collection. :-)

**Comment by robhagemans** [comment]
Thanks - personally I thought the LICENSE.md is clear enough, but let me
know if you have suggestions for improvement.

Unfortunately I can't be that much more specific as the situation can be a
bit vague and may well vary a lot by location. I.e. I'm confident what I'm
doing is OK in the locations that matter to me, but I can't advise others
as to their situation.

Getting more clarity on this is not worth while for me as it would involve
costly legal advice and is mostly of academic interest. Which unfortunately
means everyone has to weigh this up for themselves.

You'll see a similar position in others doing similar projects - e.g. see
the readme at int10h.org

**Reply by olikraus** [reply]
"int10h.org" clearly states that all fonts are available under CC BY-SA 4.0

No doubt, it is difficult and time consuming to document the font license and/or copyright information. 
I did my very best to do so, see for example here https://github.com/olikraus/u8g2/wiki/fntgrpadobex11#copyright or here https://github.com/olikraus/u8g2/wiki/fntgrpunifont#copyright
In fact all of the bitmap fonts included in my u8g2 project include download location and copyright statements.

This is the way how I handle this. I might be overcautious, but my experience is, that many users are glad to have this information. 
So.. at the moment I will add fonts to u8g2 only, if the copyright is clear and the source is known for later tracing.

> but let me know if you have suggestions for improvement.

Either A copyright statement or the source of the font (so that I can do my own research) would be great.
At least, for whatever font such a copyright statement exists, it would be great to have such hint.

Let's take the lisa fonts as example: https://github.com/robhagemans/hoard-of-bitfonts/tree/master/apple/lisa
I indeed like this font, but there is no hint regarding the original source. Are the fonts part of the download package from here: https://info.computerhistory.org/apple-lisa-code ? If so then at least I could assume that the license from that page applies.

In general it would be also nice to read more about your effort to find and restore those fonts ;-)

Anyhow ... no need to change something. I just was wondering whether I missed the copyright information, so question is answered.

**Reply by robhagemans** [reply]
I'm refering to this: https://int10h.org/oldschool-pc-fonts/readme/#legal_stuff . Like the fonts here, those fonts are used under the understanding that no copyright applies. The license is for ViLeR's own (considerable, and high-quality) work in extending the fonts beyond codepage 437 and compiling them to TrueType. So I fear you may be reading more into that than is justified.

If copyrights apply to those bitmaps in your jurisdiction, you may not have the right to use them, that licence notwithstanding. 
Note that I do exactly the same in LICENSE.md, except with a more liberal CC0 license, as I don't feel I've done any work to justify anything more - I haven't done nearly the work ViLeR has:

> Insofar as I have created copyrightable work by representing these typefaces in their present form: to the extent possible under law, I 
> waive all copyright and related or neighbouring rights to that additional work under the [Creative Commons Zero dedication]
> (http://creativecommons.org/publicdomain/zero/1.0/).

Either copyright on bitmaps applies for you, and in that case a priori you may have to assume you can't use them; or it doesn't, in which case there's just the courtesy of acknowledging any notices and names present in the files.

My main aim with this site is preserving the fonts in a way people can appreciate them. I agree that it would be nice to have more information documented on how this was extracted, but my time is finite and doing so would mean I'd get less done. It's a trade off. You'll find that many of the files do contain such information however.

If you have the original sources, there's always my other project `monobit` to extract them to a file much like the ones you find here. That's likely ll I did for this repo...

**Reply by olikraus** [reply]
Thanks for all the hints. I have looked into your references, but still have some doubts regarding the copyright.
But, as I said before, there is nothing you need to change. This project is already a great source for bitmap fonts and I know the effort which is required to setup such a great collection.
